### ISSUE TITLE
Livechat API src url incorrect

### ISSUE THREAD

**Issue by tsmgeek** [RocketChat/Rocket.Chat#15379]
Using the script shows on `installation` panel in UI the returned src url does not work and returns 404 `https://rocketchat.123456789.com/livechat/1.0.0/rocketchat-livechat.min.js` but if you use `https://rocketchat.123456789.com/livechat/rocketchat-livechat.min.js`

```
<!-- Start of Rocket.Chat Livechat Script -->
<script type="text/javascript">
(function(w, d, s, u) {
	w.RocketChat = function(c) { w.RocketChat._.push(c) }; w.RocketChat._ = []; w.RocketChat.url = u;
	var h = d.getElementsByTagName(s)[0], j = d.createElement(s);
	j.async = true; j.src = 'https://rocketchat.123456789.com/livechat/1.0.0/rocketchat-livechat.min.js?_=201903270000';
	h.parentNode.insertBefore(j, h);
})(window, document, 'script', 'https://rocketchat.123456789.com/livechat?version=1.0.0');
</script>
<!-- End of Rocket.Chat Livechat Script -->
```

### Server Setup Information:

- Version of Rocket.Chat Server: 1.3.2 
- Deployment Method: docker

**Comment by renatobecker** [comment]
Related to -> https://github.com/RocketChat/Rocket.Chat.Livechat/issues/290

Yeah, we removed the `1.0.0` from the Livechat public path URL, but it had not been removed from the installation page.
It's fixed on our latest release -> `2.0`.

Thanks.

**Comment by tsmgeek** [comment]
Thanks for pointer
